frameworks/helloworld/src/main/dist/shm.yml

```diff
@@ -0,0 +1,22 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+  user: {{FRAMEWORK_USER}}
+pods:
+  hello:
+    count: {{HELLO_COUNT}}
+    ipc-mode: PRIVATE
+    shm-size: 128
```

Comment:
another thing I would like to double check here is that - when the ipc-mode is PRIVATE, the shm-size field could still be optional.

The shm-size take precedence as follow:
1. users specified shm-size
2. If users do not specify any, Mesos takes [cluster default shm-size](https://github.com/qianzhangxa/dcos/blob/552c13891a0d63c4dacc18a0c0cba4828d62b2b7/gen/dcos-config.yaml#L1394)
3. If there is no default cluster shm size provided, Mesos takes the kernel default behavior - half size of the host memory.

so tl;dr, please double check when ipc-mode is PRIVATE, shm-size is optional.

Thank you!